packages/webpack/lib/plugins/render.js

```diff
@@ -0,0 +1,42 @@
+const indexFile = require('directory-index');
+const { RawSource } = require('webpack-sources');
+const deprecate = require('depd')('hops-webpack');
+const { trimLeadingSlash } = require('pathifist');
+
+exports.RenderPlugin = class RenderPlugin {
+  constructor(render, requests) {
+    deprecate(
+      '[DEP003] Static rendering is deprecated and will be removed in a future major release ' +
```

Comment:
`DEP003` (actually `DEP0003`…) is already reserved for the `shouldPrefetchOnServer` preset.

Also the URL to the `DEPRECATIONS.md` is wrong.

Reply:
See https://github.com/xing/hops/pull/1164#discussion_r412828641